tools/accuracy_checker/accuracy_checker/adapters/yolo.py

```diff
@@ -653,3 +653,80 @@ def process(self, raw, identifiers, frame_meta):
                 identifier, j, scores, x_mins, y_mins, x_maxs, y_maxs, meta
             ))
         return result
+
+
+class YoloxAdapter(Adapter):
+    __provider__ = 'yolox'
+    prediction_types = (DetectionPrediction, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'threshold': NumberField(value_type=float, optional=True, min_value=0, default=0.001,
+                                     description="Minimal objectiveness score value for valid detections."),
+            'output_name': StringField(optional=True, default=None, description="Name of output.")
+        })
+        return parameters
+
+    def configure(self):
+        self.threshold = self.get_value_from_config('threshold')
+        self.output_name = self.get_value_from_config('output_name')
+
+    @staticmethod
+    def boxes_postprocess(boxes, img_size):
+        grids = []
+        expanded_strides = []
+
+        strides = [8, 16, 32]
```

Comment:
there is a lot of common with yolor, is it possible to use common adapter or reuse some its parts?

Reply:
Done